src/demos/wizard/wizard-demo-modal.component.ts

```diff
@@ -34,17 +39,11 @@ export class SkyWizardDemoModalComponent {
     }
   }
 
-  public get isDone(): boolean {
-    return this.activeIndex === 2;
-  }
-
   constructor(
-    public instance: SkyModalInstance,
-    private changeDetector: ChangeDetectorRef
```

Comment:
Can you get rid of the `ChangeDetectorRef` import now?

Reply:
Should be gone now.